ng-dev/release/publish/actions.ts

```diff
@@ -221,6 +226,8 @@ export abstract class ReleaseAction {
     await this.createCommit(commitMessage, [
       workspaceRelativePackageJsonPath,
       workspaceRelativeChangelogPath,
+      // Ensure modified Aspect lock files are included in the release commit.
```

Comment:
In some cases, the pnpm lock file gets also updated, example when a package is deprecated. Do we want to handle this case too?

See: https://github.com/angular/angular-cli/pull/29175

Reply:
Do you think that was caused by e.g. doug running his own version of pnpm?

Reply:
No, I encountered I encountered this also on my machine, it think the reasoning is that the sync task cause the lock file not to be immutable.

Reply:
Likely the deprecated message was added recently?

Reply:
Ahh good point. I will add the lock file